ui/src/utils/constants.ts

```diff
@@ -207,6 +207,12 @@ export const FREQUENCY_OPTIONS = [
 export const PartitioningRegexTooltip =
 	"Choose a column to partition your data for faster reads and better performance"
 
+export const DestinationTableToolTipText =
+	"Defines the table’s appearance and its destination database where it will be stored"
+
+export const DestinationDatabaseTooltipText =
```

Comment:
```suggestion
export const DESTINATATION_DATABASE_TOOLTIP_TEXT =
```

minor: just to make naming consistent